clouddrift/wavelet.py

```diff
@@ -346,7 +367,7 @@ def morse_wavelet(
     length: int,
     gamma: float,
     beta: float,
-    radian_frequency: np.ndarray,
+    radian_frequency: float | np.ndarray,
```

Comment:
I am pretty sure the code needs the radian frequency to be an array, not a float, even with a single element.

Reply:
Got it, I will correct this.